State that ONLY error mapping can happen in the OT Shim layer. (#2148)
open-telemetry/opentelemetry-specification: specification/compatibility/opentracing.md

simultaneously in a running service, in order to ease migration from the former
to the latter.

> Observe that semantic convention mapping MUST NOT be performed, with the

Reviewer: ```suggestion
Semantic convention mapping SHOULD NOT be performed, with the
```
